### PR TITLE
Traffic objects

### PR DESCRIPTION
**Pull request by Jessime**
This pull request is addressing https://github.com/PyGithub/PyGithub/issues/478, which wants to expand PyGithub's functionality to include [traffic](https://developer.github.com/v3/repos/traffic/). I've only included Clones and Views, to get started. Referrers and Paths haven't been implemented yet, but shouldn't be hard to add.

Also, this is my second pull request ever, so my apologies if I'm doing anything incorrectly. 

Thanks!

### CODE REVIEW

File: github/Repository.py [hunk 1]
```diff
@@ -1154,6 +1155,16 @@ def get_branches(self):
             None
         )
 
+    def get_clones(self):
+        """
+        :calls: `GET /repos/:owner/:repo/traffic/clones <https://developer.github.com/v3/repos/traffic/clones>
```
Comment (sfdye) [inline github/Repository.py]: Link should be:
https://developer.github.com/v3/repos/traffic/#clones

File: github/Traffic.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+# ##############################################################################
+
+import github.GithubObject
+
+class Clones(github.GithubObject.CompletableGithubObject):
+    """Represents clone traffic for a repository
+    
+    https://developer.github.com/v3/repos/traffic/clones
```
Comment (sfdye) [inline github/Traffic.py]: Same here.

File: github/Traffic.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    def _useAttributes(self, attributes):      
+        if "count" in attributes:  # pragma no branch
+            self._count = self._makeIntAttribute(attributes["count"])
+        if "uniques" in attributes:  # pragma no branch
+            self._uniques = self._makeIntAttribute(attributes["uniques"])
+        if "clones" in attributes:  # pragma no branch
+            self._clones = self._GithubObject__makeSimpleListAttribute(attributes["clones"], dict)
+            
+
+
```
Comment (sfdye) [inline github/Traffic.py]: Looks like you got some extra lines here.

File: github/Traffic.py [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        if "clones" in attributes:  # pragma no branch
+            self._clones = self._GithubObject__makeSimpleListAttribute(attributes["clones"], dict)
+            
+
+            
+
+class Views(github.GithubObject.CompletableGithubObject):
+    """Represents view traffic for a repository
+    
+    https://developer.github.com/v3/repos/traffic/clones
```
Comment (sfdye) [inline github/Traffic.py]: Here too.

File: github/Traffic.py [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+    def _useAttributes(self, attributes):      
+        if "count" in attributes:  # pragma no branch
+            self._count = self._makeIntAttribute(attributes["count"])
+        if "uniques" in attributes:  # pragma no branch
+            self._uniques = self._makeIntAttribute(attributes["uniques"])
+        if "views" in attributes:  # pragma no branch
+            self._views = self._GithubObject__makeSimpleListAttribute(attributes["views"], dict)
+            
+            
+
```
Comment (sfdye) [inline github/Traffic.py]: Extra lines at the end.

**Comment by sfdye** [comment]
Hi @Jessime thanks for sending your PR. Would you mind adding the unit tests for it?

**Comment by Jessime** [comment]
Thanks for the suggestions. I've updated all of the extra lines and bad links. I've started working on unit tests, but I don't have much experience here and have gotten stuck. Specifically, I don't know what the proper procedure is for generating and storing data to test against.

**Comment by sfdye** [comment]
@Jessime A good way to start is to look at the existing test and replay data.

**Comment by sfdye** [comment]
@Jessime Please have a look at the docs: https://github.com/PyGithub/PyGithub/blob/master/CONTRIBUTING.md#automated-tests

Let me know if this makes sense or not.

**Pull request closed without merging**